components/layout/SideNav.tsx

```diff
@@ -241,7 +262,12 @@ const DocGroup = ({ title, docs }: DocGroupProps) => {
 
   return (
     <>
-      <GroupToggle onClick={() => setIsExpanded(!isExpanded)}>
+      <GroupToggle
+        tabIndex={0}
+        onClick={() => setIsExpanded(!isExpanded)}
+        onKeyPress={(e) =>
+          e.key === "Enter" ? setIsExpanded(!isExpanded) : false}
+      >
```

Comment:
Made this focusable so the links sorted under them could be accessed via keyboard